src/main/scala/uk/gov/nationalarchives/tdr/api/service/ValidateFileMetadataService.scala

```diff
@@ -18,21 +19,6 @@ class ValidateFileMetadataService(
     fileStatusRepository: FileStatusRepository
 )(implicit val ec: ExecutionContext) {
 
-  def toPropertyNames(fields: Seq[CustomMetadataField]): Set[String] = fields.map(_.name).toSet
-
-  def toAdditionalMetadataFieldGroups(fields: Seq[CustomMetadataField]): Seq[FieldGroup] = {
-    val closureFields = fields.filter(f => f.propertyGroup.contains("MandatoryClosure") || f.propertyGroup.contains("OptionalClosure"))
-    val descriptiveFields = fields.filter(f => f.propertyGroup.contains("OptionalMetadata"))
-    Seq(FieldGroup(ClosureMetadata, closureFields), FieldGroup(DescriptiveMetadata, descriptiveFields))
-  }
-
-  def toValueDependenciesGroups(field: CustomMetadataField): Seq[FieldGroup] = {
```

Comment:
Is this method no longer needed? I can't see anymore in the refactored code?

Reply:
Yeah, these are unused methods.